tests/CMakeLists.txt

```diff
@@ -140,6 +141,8 @@ add_test(NAME launcher_test COMMAND tests/launcher_test.py -v --clean-run WORKIN
 set_property(TEST launcher_test PROPERTY LABELS nonparallelizable_tests)
 add_test(NAME keosd_auto_launch_test COMMAND tests/keosd_auto_launch_test.py WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
 set_property(TEST keosd_auto_launch_test PROPERTY LABELS nonparallelizable_tests)
+add_test(NAME nodeos_snapshot_diff_test COMMAND tests/nodeos_snapshot_diff_test.py -v --clean-run --dump-error-detail WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
```

Comment:
`--dump-error-detail` should be removed. It is not needed any more.

Reply:
Thanks for catching that.